packages/bruno-app/src/utils/collections/index.js

```diff
@@ -417,7 +417,9 @@ export const transformRequestToSaveToFilesystem = (item) => {
   });
 
   if (itemToSave.request.body.mode === 'json') {
-    itemToSave.request.body.json = replaceTabsWithSpaces(itemToSave.request.body.json);
+    const copyItemToSave = cloneDeep(itemToSave);
```

Comment:
Actually, in my last comment I meant cloning the items that are received in the `saveMultipleRequests` function. Although doing the change here also solves the bug, I believe that applying this fix there can avoid this bug arising from a similar mutation in another function or even in another part of `transformRequestToSaveToFilesystem`.

Reply:
I've made this comment as a suggestion of improvement, but I don't actually have the necessary permissions to approve it and merge it. For that we relies in @helloanoop